tools/mcsema_lift/Lift.cpp

```diff
@@ -117,3 +121,65 @@ int main(int argc, char *argv[]) {
 
   return EXIT_SUCCESS;
 }
+
+namespace {
+
+static void PrintVersion(void) {
+  std::cout
+      << "This is mcsema-lift version: " << MCSEMA_VERSION_STRING << std::endl
+      << "Built from branch: " << MCSEMA_BRANCH_NAME << std::endl
+      << "Using LLVM " << LLVM_VERSION_STRING << std::endl;
+}
+
+static void DumpCFG(const std::string &cfg_path) {
```

Comment:
Can you make this take a `const NativeModule *`, and then call `DumpCFG` in `main` after reading the protobuf, and perhaps moving that code up before finding the semantics bitcode file.